src/TokenizedStrategy.sol

```diff
@@ -2014,7 +2014,7 @@ contract TokenizedStrategy {
                     keccak256(
                         "EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)"
                     ),
-                    keccak256(bytes(S.name)),
```

Comment:
I really know nothing about this, but is there a reason make this an exact match against the multi-strat vaults or not?